man/tpm2_rc_decode.1.md

```diff
@@ -0,0 +1,46 @@
+tpm2_rc_decode 1 "SEPTEMBER 2017" tpm2-tools
+==================================================
+
+NAME
+----
+
+tpm2_rc_decode(1) - Decode TPM2 error codes to human readable format.
+
+SYNOPSIS
+--------
+
+`tpm2_rc_decode` [OPTIONS] _RC\_CODE_
```

Comment:
Wonder if makes sense to have an [OPTIONS] for tools that don't have options (this comment applies to other tools as well like tpm2_nvlist).

Reply:
Ah, forget this comment. Of course it makes sense since we still have the common and TCTI options.